### PR TITLE
Marian's Blog Post

### PR DESCRIPTION
**Pull request by marianagyby**
This PR is to add Marian's blog post called "With Great Data Science Comes Great Responsibility" to the MDS website.

### CODE REVIEW

File: _posts/with-great-data-science.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# With Great Data Science Comes Great Responsibility
```
Comment (phchen5) [inline _posts/with-great-data-science.md]: Please add the following to the top of the file:

```suggestion
---
layout: post
title: With Great Data Science Comes Great Responsibility
subtitle: by Marian Agyby
---
```

File: _posts/with-great-data-science.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+<p align="center">
+  <img src="../img/blog/img_with_great_ds/img3.png" alt="img3">
+</p>
+
+<p align="center">
+  <a href="https://aspectmr.com/misleading-graphs/">(Source)</a>
+</p>
+
+---
+
```
Comment (phchen5) [inline _posts/with-great-data-science.md:104]: At the end of the blog post, include a short description/link of yourself

```suggestion
### Authors:
[Author Name](link-to-your-bio) is {brief description of yourself}
```

File: _posts/with-great-data-science.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
+# With Great Data Science Comes Great Responsibility
+### Warning signs and tools to stop the spread of misinformation.
+#### By Marian Agyby
```
Comment (phchen5) [inline _posts/with-great-data-science.md]: This can be removed since it's mentioned above
```suggestion
```

**Review by phchen5 — COMMENTED** [review]
THanks Marian! Just a few minor comments.

**Comment by kvarada** [comment]
LGTM. Thank you @marianagyby! I'm merging the pull request.

**Pull request merged**